### ISSUE TITLE
Crash when used with Ballistix, ElectroDynamics, Nuclear Science (unknown issue)

### ISSUE THREAD

**Issue by JankGamer** [HellFirePvP/AstralSorcery#1935]
Playing through ATM6 pack (1.16.5) with no issues. Added in the mods in the title (although, think its ElectroDynamics causing it) and the pack crashes on opening load. Crash log (https://pastebin.com/DREdCSM1) refers to hellfire/astral sorcery and reliquary but I'm not sure the issue lies with these mods. Asked on Forge, was directed to ask Astral/Reliquary mod team as they are the ones referenced.

Any help in navigating would be much appreciated!

**Comment by PocketfulOfGeese** [comment]
After a quick look, looks like it's reliquary throwing an error because of a capability added for the Resplendent Prism from astral sorcery. Reliquary team might be able to provide further information.

**Comment by JankGamer** [comment]
Yeah, I've asked there too. My issue is that the two play entirely (like zero issues) fine together when the aforementioned mods are not included in the mix. Fine before, fine after, not fine when they are present.
